psi4/src/psi4/lib3index/dfhelper.cc

```diff
@@ -1189,14 +1314,18 @@ double* DFHelper::metric_prep_core(double pow) {
             break;
         }
     }
+    timer_on("DFH: metric power");
     if (!on) {
         power = pow;
-        timer_on("DFH: metric power");
         SharedMatrix J = metrics_[1.0];
-        J->power(power, condition_);
+        if (fabs(pow + 1.0) < 1e-13) {
+            J->invert();
+        } else {
+            J->power(power, condition_);
+        }
         metrics_[power] = J;
-        timer_off("DFH: metric power");
     }
+    timer_off("DFH: metric power");
```

Comment:
I think the on/off was outside the if so that no entry appeared if off. any reason to change that?

Reply:
I added that timer to do some debugging, so I'll just get rid of it.